### ISSUE TITLE
The WM_Sn selection is acquired incorrectly (the code actually acquires WM_S_Sn)

### ISSUE THREAD

**Issue by psychon** [awesomeWM/awesome#3561]
See https://github.com/i3/i3/pull/4843

The TL;DR version: This code should use `"WM"` instead of `"WM_S"`: https://github.com/awesomeWM/awesome/blob/2d6244f9445ffa22c8d53b3e4b4641e0e5bb6170/awesome.c#L288

**Comment by Aire-One** [comment]
How do we test that in awesome? I mean, I don't think we have any test in this area...

**Comment by psychon** [comment]
No idea how to test this, sorry. :shrug: 

Hm, actually... I guess this bug means that `--replace` does not work correctly. AwesomeWM should be able to replace AwesomeWM, but the interaction with other WMs should be broken. So, you could start `Xephyr :2`, then run `DISPLAY=:2 awesome` and `DISPLAY=:2 some-other-wm --replace`. (Or the other way around, first another WM and then AwesomeWM.)

But this is just a guess and also not helpful for an integration test.

**Comment by actionless** [comment]
aforementioned i3 is just checking for wm hint to be present?

**Comment by psychon** [comment]
@actionless Yeah, pretty much. It uses perl xcb bindings to send a `GetSelectionOwner` request and checks that the reply's `id` member is not 0 (`!= XCB_NONE`).
